sass/cd/cd-footer/_cd-mandate.scss

```diff
@@ -30,7 +30,7 @@
 .cd-mandate__text {
   padding: 0 $cd-container-padding;
   max-width: 540px;
-  margin: 0 auto;
+  margin: 0 auto 1em;
```

Comment:
whoops I slid this into the branch on accident instead of splitting out into another PR. sorry!